manuscript.md

```diff
@@ -37,104 +37,81 @@ We will also explore how other omics fields are dealing with OSS and FAIR softwa
 
-In proteomics, Percolator (https://github.com/percolator/percolator) is an open-source project with more than 700 citations and 20 contributors [17952086]; and a core tool for multiple open-source tools [], commercial software [], and pipelines [].
-The idea that open-source software is of lower quality is a myth that has been debunked by the success of many open-source projects in the field of computational proteomics and bioinformatics. [If it was debunked why does Proteomics still have this problem? Why do we write this manuscript? I guess you can demonstrate the success of Genomics and the healthy software ecosystem there - and contrast it with Proteomics or Metabolomics] [M: the community thinks academic proteomics software is of lower quality ...  https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00015]
+In addition to all these misconceptions and complexity, journals mandate code availability as a publication requirement, which has prompted multiple bad practices from software developers and bioinformaticians aiming to futfill the journal's requirements. 
 
-[W: I moved this paragraph down because I think it makes more sense to first discuss different aspects of OSS, and afterwards transition to closed source.]
-In contrast, closed source means that the source code of the software is not published and it is not shared with the public for anyone to look at or change. [True, but it can also be public, but not allowed to be modified or shared.]
-A common misconception is that closed-source software with "academic" licenses are equivalent to open-source. [W: What do you understand under academic licenses? Do you mean that the code is open to academics, or just that academics can run the software for free / at reduced cost? This should be defined for the reader.] [B: usually this refers to free-for-academic-use only. And this is bullshit, because no-one can define academics.]
+- Researchers upload closed-source software to platforms like GitHub, giving an impression of openness with features such as issue tracking, while the actual source code remains inaccessible. Although often well-intentioned, this practice can mislead other scientists and, in our view, should be discouraged. 
+- Software is deposited in GitHub as open-source during the submission of the manuscript, but after publication; software licenses in GitHub repositories are changed over time, and repositories themselves can be deleted or made private, complicating efforts to ensure long-term accessibility and reliability.
```

Comment:
```suggestion
- Software is deposited in GitHub as open-source during the submission of the manuscript, but after publication; software licenses in GitHub repositories are changed, repositories deleted or made private, complicating efforts to ensure long-term accessibility.
```

Reply:
Maybe something like this?

"Software deposited on GitHub as open-source during the submission can change its license after publication of the manuscript, or even worse, repositories get deleted or made private. These steps complicate efforts to ensure long-term accessibility to the code."